RM_SUI/RM_SUI/View/VerificationView.swift

```diff
@@ -0,0 +1,196 @@
+//
+//  VerificationView.swift
+//  RM_SUI
+//
+//  Created by Дмитрий Супрун on 24.01.23.
+//
+
+import SwiftUI
+
+/// Verification login
+struct VerificationView: View {
+    // MARK: - Private Constants
```

Comment:
```suggestion
    // MARK: - Private Constants
    
```
Или форматер поставь или сделай везде был один стиль (если после марки есть пробел - то везде есть)